filament/src/PostProcessManager.cpp

```diff
@@ -979,7 +975,7 @@ void PostProcessManager::colorGradingSubpass(DriverApi& driver,
     Handle<HwRenderPrimitive> const& fullScreenRenderPrimitive = engine.getFullScreenRenderPrimitive();
 
     FMaterialInstance* mi = mColorGradingAsSubpass.getMaterialInstance();
-    mi->setParameter("lut", mTonemapper->getHwHandle(), {
+    mi->setParameter("lut", mEngine.getDefaultColorGrading()->getHwHandle(), {
```

Comment:
This will come from the View eventually, right?

Reply:
That's correct, this is temporary. We will still use this default color grading by default on Views though.